.github/workflows/ci.yml

```diff
@@ -216,7 +216,7 @@ jobs:
         run: yarn changeset version --snapshot
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-      - name: Build package
+      - name: Build packages
         run: yarn build:release
```

Comment:
I dont think building `build:release` is needed here since we have `need: [build-release]` meaning packages are build on CI already, am I correct @FRSgit , if so, same goes into production process